aces/Loris
# [Statistics] updated module help text

Updated copy for the Statistics module help text

#### Testing instructions

1. Review copy for accuracy, etc.
2. See screenshot below for rendering in LORIS

#### Links to related tickets

* https://github.com/aces/Loris/issues/5576

![Screenshot 2019-11-14 13 37 07](https://user-images.githubusercontent.com/37813606/68885811-e191e800-06e3-11ea-9320-6c4dce0210aa.png)
